src/openms/source/ANALYSIS/QUANTITATION/PeptideAndProteinQuant.cpp

```diff
@@ -431,7 +431,7 @@ namespace OpenMS
     {
       if (proteins.getHits().empty())
       {
-        OPENMS_LOG_ERROR << "No protein matches found, cannot calculate iBAQ!";
+        OPENMS_LOG_FATAL_ERROR << "No protein matches found, cannot calculate iBAQ!";
```

Comment:
add << std::endl;
Does this throw an exception / terminate the program?

Reply:
It should: "Macro to be used if fatal error are reported (processing stops)"

Reply:
can you double check? the documentation could just mean that this needs to be used if processing is stopped.
Ambiguous right now

Reply:
I tried to test it, but it seems that in a standard workflow this will not be reached. I've added it to address an issue in the class test, where no protein hits were set. However, it seems, that this exception has already been catched.

Reply:
Are you sure that an exception is thrown?

Reply:
for me it looks like it is just a generic log stream that outputs to cerr by default:
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/CONCEPT/LogStream.cpp#L565

Reply:
But when I remove the protein references in all peptide entries, the program runs through. However, the results are then empty.

Reply:
can we zoom?